### ISSUE TITLE
FDR Panic on A12

### ISSUE THREAD

**Issue by httpwemel** [futurerestore/futurerestore#36]
Downgrading to iOS 14.5.1 using iOS 15.1 beta 1 SEP + Baseband (3.50.9 Baseband which should be compatible theoretically and is still signed) on iPhone XR with Blobs but panics with FDR timeout. Using Nightly Build (Oct 8, 2021)

```
Sending NORData now...
Done sending NORData
FDR 0x7fdb4f0f1170 timeout waiting for command
FDR 0x7fdb4f0f1170 waiting for message...
idevice_event_cb: device 00027d800a32002e (udid: 00008020-00027D800A32002E) disconnected
ERROR: Unable to receive message from FDR 0x7fdb4f0f1170 (-2). 0/2 bytes
FDR 0x7fdb4f0f1170 terminating...
ERROR: Could not read data (-256). Aborting.
Cleaning up...
[exception]:
what=ERROR: Unable to restore device

code=101384209
line=1547
file=futurerestore.cpp
commit count=253:
commit sha  =b49aad07f19b0b17d4ae97ab89675c971747b989:
Done: restoring failed!
httpwemel@Wemels-MacBook-Pro futurerestore %
```

**Comment by Cryptiiiic** [comment]
@httpwemel you will lose faceid and rootfs will not be working quite right, I can enable a12 but I don't recommend it. Restore  won't even complete it will just fdr loop infinitely, when you reboot and exit recovery, rootfs will be improper and faceid will be not working.

**Comment by Cryptiiiic** [comment]
Fixed in 5c6f862
